composer.json

```diff
@@ -11,15 +11,15 @@
   "license": "AGPL-3.0-or-later",
   "type": "project",
   "require": {
-    "php": "^7.3",
+    "php": ">=7.4 <8.1",
     "ext-curl": "*",
     "ext-fileinfo": "*",
     "ext-json": "*",
     "ext-mbstring": "*",
     "ext-pdo": "*",
     "adldap2/adldap2": "^10.2",
     "alek13/slack": "^2.0",
-    "bacon/bacon-qr-code": "^1.0",
+    "bacon/bacon-qr-code": "^2.0",
```

Comment:
This one looks pretty innocuous - nothing deprecated, minimum PHP version is 7.1. Should be fine?

Reply:
Used by our google two-factor auth stuff (only indirectly)